src/Dashboard.tsx

```diff
@@ -93,10 +93,10 @@ class Dashboard extends React.PureComponent<{}, {}> {
             <i>This is a mockup</i>
             <Header size="medium">Recently viewed persons</Header>
             <Card.Group>
-               <PersonCard name="Emmanuel Briot" dates="1975-" />
-               <PersonCard name="John Smith" dates="1950-2008" image="/assets/roland.jpg" />
-               <PersonCard name="Louis XIII" image="/assets/jacques_LE_TEXIER.jpg" />
-               <PersonCard name="Josephine" image="invalid"/>
+               <PersonCard name="Emmanuel Briot" dates="1975-" image="/assets/Emmanuel_Briot.jpg"/>
+               <PersonCard name="John Smith" dates="~January, 1580-June 21, 1631" image="/assets/John_Smith.jpg" />
+               <PersonCard name="Louis XIII" dates="September 27, 1601-May 14, 1643" image="/assets/Louis_XIII.jpg" />
+               <PersonCard name="Josephine" dates="June 23, 1763-May 29, 1814" image="/assets/Josephine.jpg"/>
```

Comment:
I had left the "Josephine" card with an invalid image and no dates on purpose. These are data that might not be available for some persons, and the cards should still work in that context. Can we still have some nice layout even if there's no image (which is likely to be the vast majority of persons in practice) ? For the same reason, I had a card with an image with no date, though I realize I was missing a card with no image but with dates.

If you are interested in design, would you like to experiment with this ?

Reply:
Also, my image can't be mine, I don't have a twitter profile :-) You can use the one from github, or you can use any random image from the web, I don't mind !